purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
+    package_id = IntegerProperty(required=True)
+    task_id = IntegerProperty(required=True)
+    build_id = IntegerProperty()
+    name = StringProperty()
+    owner_name = StringProperty()
+    package_name = StringProperty(required=True)
+    version = StringProperty(required=True)
+    release = StringProperty(required=True)
+    epoch = StringProperty()
+    nvr = StringProperty()
+    source = StringProperty()
+    state = IntegerProperty(required=True)
+    volume_name = StringProperty()
+    start_time = DateTimeProperty()
+    creation_time = DateTimeProperty(required=True)
+    completion_time = DateTimeProperty()
+    start_ts = FloatProperty()
```

Comment:
Remove either the "ts" or the "time" version of this so we only have one.  Pick whichever is most natural.